core/config/docs/core.toml

```
@@ -509,6 +509,26 @@ DeltaReconcile = '1m' # Default
 # but the host and port must be fully specified and cannot be empty. You can specify `0.0.0.0` (IPv4) or `::` (IPv6) to listen on all interfaces, but that is not recommended.
 ListenAddresses = ['1.2.3.4:9999', '[a52d:0:a88:1274::abcd]:1337'] # Example
 
+[Capabilities.GatewayConnectorConfig]
+# ChainIDForNodeKey is the ChainID of the network
+ChainIDForNodeKey = '0xfd29Dd9C980D715a64dace97F7A2AB98bcaE0fed' # Default
```

The default here is an integer such as 11155111 for Ethereum Sepolia, not a hex address.

---

But still a string to be agnostic? And this should be an example, no?
```suggestion
ChainIDForNodeKey = '11155111' # Example
```

---

Yes, a string, thanks